docs/user/alerts.md

```diff
@@ -60,6 +61,28 @@ the Machine API resources to discover the root cause. For more information on
 why nodes, or machines, might not become ready please see the
 [Machine API FAQ](https://github.com/openshift/machine-api-operator/blob/master/FAQ.md).
 
+## ClusterAutoscalerExcessiveUnregisteredNodeCleanup
+The cluster autoscaler has removed more than 1 node that failed to register within the
+last 6 hours. This indicates that that the autoscaler has been replacing nodes which
+are failing to join the cluster.
+
+### Query
+```
+rate(cluster_autoscaler_old_unregistered_nodes_removed_count[6h]) > 1
+```
+
+### Possible Causes
+* MachineSet, or cloud credential, configuration is causing machines to fail.
+* Network disruption is preventing machines from becoming nodes.
+
+### Resolution
+* Look at the cluster autoscaler events, or logs, to determine which Machines have been removed
+  and to which MachineSets they belong.
```

Comment:
Can we make the MachineSet name a label on the metric, and then include that name in the alert to make running this down a bit more convenient for folks who are triaging it?

Reply:
i think that's a great idea, but upstream did not like having a label with unbounded cardinality on the metric. i don't think it would be an issue in production, but i can understand their point so we decided to drop the label in favor of having the metric.